webapp/locales/en.json

```diff
@@ -329,6 +329,28 @@
       "mute": "Mute user",
       "unmute": "Unmute user",
       "unmuted": "{name} is unmuted again"
+    },
+    "blocked-users": {
+      "name": "Blocked users",
+      "explanation": {
+        "intro": "If another user has been blocked by you, this is what happens:",
+        "your-perspective": "You will no longer be able to interact with their contributions.",
+        "their-perspective": "Vice versa: The blocked person will also no longer be able to interact with your contributions.",
+        "notifications": "Blocked users will no longer receive notifications if they mention each other.",
```

Comment:
Is that actually true? I suggested to remove it from the German translations, see above :point_up:

Reply:
yes, see my response :point_up_2: